buildSrc/subprojects/configuration/src/main/kotlin/org/gradle/gradlebuild/dependencies/DependenciesMetadataRulesPlugin.kt

```diff
@@ -52,6 +52,9 @@ open class DependenciesMetadataRulesPlugin : Plugin<Project> {
                 withModule("org.jmock:jmock-legacy", ReplaceCglibNodepWithCglibRule::class.java)
                 withModule("cglib:cglib", NoAntRule::class.java)
 
+                // asciidoctorj depends on a lot of stuff, which causes `Can't create process, argument list too long` on Windows
+                withLibraryDependencies("org.gradle:sample-discovery", DependencyRemovalByNameRule::class, setOf("asciidoctorj", "asciidoctorj-api"))
```

Comment:
I guess excluding these dependencies from `exemplar` is not necessary anymore after the upgrade: https://github.com/gradle/gradle/pull/6682/files#diff-dbfb317d8316634875a188ff15b832e9R15

Reply:
Same here: https://github.com/gradle/gradle/pull/6682/files#diff-9fd7ac19f0583dbdcec4fec7a8fbff3bR43

Reply:
Why? #6682 is just excluding `groovy` and `slf4j`, it doesn't touch `asciidoctorj` at all. I rebase current `master` to this branch and remove this line, `asciidoctorj` does appear in the dependencies.

Reply:
I am saying the excludes are not needed anymore with the new exemplar version.

Reply:
Yes, new exemplar won't depend on `asciidoctorj` by default. But I don't understand why you pasted #6682 , did #6682 perform any upgrade of exemplar?

Reply:
Ah I understand what you're saying, yeah, excluding these stuff won't be necessary any more. :-)